src/custom-types.json

```diff
@@ -805,7 +805,11 @@
   },
   "text/x-java-properties": {
     "compressible": true,
-    "extensions": ["properties"]
+    "extensions": ["properties"],
+    "sources": [
+      "https://stackoverflow.com/a/28813973",
```

Comment:
Thank you. These links need to be a [primary source](https://en.wikipedia.org/wiki/Primary_source) (https://github.com/jshttp/mime-db#contributing).